sites/native-twin-docs/src/components/LinkNavbar/index.tsx

```diff
@@ -0,0 +1,13 @@
+import tailwindClasses from "../../../data";
```

Comment:
maybe you need to set an alias path in your tsconfig in order to prevent long import paths